extensions/container-image/container-image-docker-common/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-container-image-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-container-image-docker-common-parent</artifactId>
+    <name>Quarkus - Container Image - Docker - Common</name>
+    <packaging>pom</packaging>
+    <modules>
+        <module>deployment</module>
+        <module>runtime</module>
+        <module>container-image-docker</module>
+        <module>container-image-podman</module>
```

Comment:
What's the rationale for having these **under**  common? It's not something I would expect to see if I opened the directory for the first time

Reply:
`container-image-podman` can be on the same level as the others

Reply:
Just so I'm clear, you mean `container-image-podman`, `container-image-docker`, and `container-image-docker-common` should all be on the same level as the others, and `container-image-podman` and `container-image-docker` should include `container-image-docker-common` as a dependency?

Reply:
yup

Reply:
Will do. I'll make this change. Sorry about that! I just assumed when one extension extends another it implies hierarchical.

Reply:
np :)

Reply:
Done.